README.md

```diff
@@ -37,9 +37,7 @@ Hvis feltet efterlades tomt, betragtes evalueringen som ikke færdiggjort.
 |  | F4  | Der er tænkt på sikkerheden i løsningen                                                                                                      | Hvordan?                                                            | sandkasse |
 |  | F5  | Løsningens formål og værdi er beskrevet                                                                                                      | Hvor?                                                               | sandkasse |
 |  | F6  | Kildekoden er overdraget og er placeret under OS2's github                                                                                      | Indsæt link                                                        | 1         |
-|  | F7  | Driftskrav til løsningen er dokumenteret                                                                                                       | Hvor?                                                               | 1         |
-|  | F8  | Løsningen er dokumenteret på teknisk niveau                                                                                                   | Hvor?                                                               | 1         |
-|  | F9  | Teknisk implementering af løsningen er dokumenteret                                                                                            | Hvor?                                                               | 1         |
+|  | F7  | Alt dokumentation til projektet udarbejdes med og overholder OS2s standardskabelon for dokumentation.                                                                                                       | [Documentation template for OS2 projects](http://github.com/OS2offdig/os2-docs-template)                                    
```

Comment:
Does this mean that we need a replacement demand for level 1 products as the demand for operating requirements being documented is deleted?

Reply:
We could add a section to the documentation template that requires a description of how you install and operate the product

Reply:
Work has started here: https://github.com/OS2offdig/os2-docs-template/issues/12